dhis-2/dhis-support/dhis-support-external/src/main/java/org/hisp/dhis/external/conf/ConfigurationKey.java

```diff
@@ -91,7 +91,14 @@
     LOGGING_ADAPTER_CONSOLE_FORMAT( "logging.console.format", "TEXT" ),
     LOGGING_FILE_MAX_SIZE( "logging.file.max_size", "100MB" ),
     LOGGING_FILE_MAX_ARCHIVES( "logging.file.max_archives", "0" ),
-    SERVER_BASE_URL( "server.base.url", "", false );
+    SERVER_BASE_URL( "server.base.url", "", false ),
+    // MONITORING //
+    MONITORING_PROVIDER( "monitoring.provider", "prometheus" ),
```

Comment:
The MONITORING_PROVIDER key is currently unused. I figured that since we use Micrometer and we can potentially export to any monitoring middleware, we can switch to different providers. I reckon it may be confusing.